ci/scripts/shipit

```diff
@@ -81,10 +81,10 @@ cat >> ${RELEASE_ROOT}/notes.md <<EOF
 ### Deployment
 \`\`\`yaml
 releases:
-- name: $RELEASE_NAME
-  version: $VERSION
-  url: https://github.com/${GITHUB_OWNER}/${RELEASE_NAME}-boshrelease/releases/download/v${VERSION}/${RELEASE_NAME}-${VERSION}.tgz
-  sha1: $SHA1
+- name: "$RELEASE_NAME"
+  version: "$VERSION"
+  url: "https://github.com/${GITHUB_OWNER}/${RELEASE_NAME}-boshrelease/releases/download/v${VERSION}/${RELEASE_NAME}-${VERSION}.tgz"
+  sha256: "sha256:$SHA256"
```

Comment:
We shouldn't replace sha1 with sha256.
Rather, we should add sha256 since sha1 might be useful for other consumers.

Reply:
Changing the key to `sha256` from `sha1` was a mistake. I will check where this is used and if that supports a `sha256` key as well.

If it does I will put both, if not I will put the syntax similar to what we have in the artifacts.yml file, i.e.:

```yaml
   sha1: "sha256:${SHA256}"
```

Reply:
The key `sha256` is not supported. A sha256 hash can only be provided via the syntax as mentioned above, with the `sha256:` prefix.

All public bosh releases we found use `sha1`, so we add the `sha256:` notation in a commented out field for those users that may benefit from it but keep the `sha1` hash as default.